### PR TITLE
Add support for iOS 15, according to Xcode 13 beta 1.

### PR DESCRIPTION
**Pull request by chrisvasselli**
Also update the project readme to explain how to make updates for the latest iOS release.

### CODE REVIEW

**Comment by chrisvasselli** [comment]
@nrbrook would you mind taking a look at this when you have a chance?

File: README.md [hunk 1]
```diff
@@ -55,6 +55,34 @@ No. The status bar server is blocked on devices. However, macOS includes the fac
...
+3. Find `UIStatusBarServerListener` in the output. 
+
+### Update the structs in the new overrider.
+
+There are two structs, StatusBarRawData and StatusBarOverrideData, that need to be updated. Each correspond to a line in the runtime header output for UIStatusBarServerListener. It should be pretty easy to figure out the mapping if you study it for a minute.
+
+1. Update StatusBarRawData and StatusBarOverrideData to match any changes to the structs in the runtime headers.
+
+### Check if it works
+That should be it!
```
Comment (daveverwer) [inline README.md:81]: Does the Markdown parser handle a heading without a newline after? It probably does, but just in case.

File: README.md [hunk 2]
```diff
@@ -55,6 +55,34 @@ No. The status bar server is blocked on devices. However, macOS includes the fac
 
 The best idea is to check [the source code](https://github.com/shinydevelopment/SimulatorStatusMagic/blob/master/SDStatusBarManager/SDStatusBarManager.m) which should get you started with how it works :)
 
+## Updating for new versions of iOS
+There's a general pattern for updating this project to support a new version of iOS, although this could change if Apple changes something in the future.
+
+### Prepare the new files.
+1. Copy the previous releases's `SDStatusBarOverriderPostXX_Y.{h,m}` files, and update them to the new version.
```
Comment (daveverwer) [inline README.md:62]: releases' 😰

File: README.md [hunk 3]
```diff
@@ -55,6 +55,34 @@ No. The status bar server is blocked on devices. However, macOS includes the fac
...
+ 1. Download the latest version of the [dsdump](https://github.com/DerekSelander/dsdump) tool.
+ 2. Run dsdump against the UIKitCore framework binary, to generate the private runtime headers.
+ ```
+ ./dsdump --objc -a x86_64 --verbose=5 /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/PrivateFrameworks/UIKitCore.framework/UIKitCore --defined > ~/Desktop/UIKitCore.txt
+ ```
+3. Find `UIStatusBarServerListener` in the output. 
+
+### Update the structs in the new overrider.
+
+There are two structs, StatusBarRawData and StatusBarOverrideData, that need to be updated. Each correspond to a line in the runtime header output for UIStatusBarServerListener. It should be pretty easy to figure out the mapping if you study it for a minute.
```
Comment (daveverwer) [inline README.md:76]: should this be "corresponds"?

**Review by daveverwer — APPROVED** [review]
What a great idea to put this documentation into the README. 🚀🚀🚀

Just a couple of minor grammar things for the review.

**Comment by daveverwer** [comment]
Let me know if/when you want to do a pod release.

**Pull request merged**